### DISCUSSION TITLE
I need help for the Extension Launcher

### DISCUSSION THREAD

**Discussion by Vhirttuegaming** [3kh0/ext-remover#833] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
It says the extension is blocked by the admin...

**Comment by AshtonDavies** [comment, marked as answer]
I think that's because the extension launcher's purpose is to get around the webstore being blocked, so it does not actually unblock extensions if you have an Allowlist.

**Comment by Aka-but-nice** [comment]
Please don't speak of this exploit any further, I hate when people mention this due to how I made it at the time. Yes, I can always create a new script but its worthless since most people have bookmarklets disabled (yes, DNS spoofing is possible but I don't care).

**Reply by AshtonDavies** [reply]
I've modified the code and made a DNS for it anyway, because people really wanted one.

It's okay for people to speak of your projects. The Extension Launcher has been shared and people want to use it and ask questions. That doesn't mean you need to stop them.

**Reply by Aka-but-nice** [reply]
Yeah, I just prefer otherwise. I've been taunted many times for it, which is why I don't like people discussing about it. And of course, I fully understand that some people agree with what you made, and some people do not, but I have realized that after all it's a useless exploit. I mean you have to think, I posted it in TN (when I was in), and people liked it, it's just that no one had the exploit work for them (most of the problems were just bookmarklet being restricted) :moyai:. In all, if you want to talk about it, you can, I just prefer if you would not.

**Reply by AshtonDavies** [reply]
Yeah, I understand it's useless but apparently some people are able to use it. I agree that it's a mid exploit, just like Dextensify and Buypass.

**Reply by Aka-but-nice** [reply]
Even Dextensify or Buypass is better than this imo 💀.

**Reply by AshtonDavies** [reply]
True 💀

**Reply by Epicminer256** [reply]
I don't think getting embarrassed of bad code or offended by criticism will make the project go anywhere. 

I think the user should be aware that there are known issues with it. Wish I wasn't busy, or I would work on that.

**Reply by Aka-but-nice** [reply]
I could easily make one right now, it's just the fact how there's almost no point for such a project.

**Reply by AshtonDavies** [reply]
Not worth running a near-pointless server just so it can go down in a few months probably.

**Reply by Aka-but-nice** [reply]
?

**Reply by AshtonDavies** [reply]
Assuming you would be hosting your DNS manually, it would come down eventually.

**Reply by Epicminer256** [reply]
I don't see a reference to a server, are we talking about the same program (ext-launcher-bookmarklet.js)?

**Reply by Aka-but-nice** [reply]
No. A DNS server in this case (this mf think it's discord :skull:).

**Reply by Epicminer256** [reply]
My B, there is no reference of it anywhere so I didn't know. If I had a clue what the original server did I would help I work with DNS poisioning with HTTP code injecting, but I don't so imma hop out this discussion. If I had anything like a bind9 config, http server, or just anything I would be interested.

**Reply by Aka-but-nice** [reply]
Yeah, it is possible and has already been done before. And regardless, unless you mean the extension launcher (which is practically useless), spoofing the webstore won't be that much of a benefit since most APIs are useless (including `management`).
